src/stories/tests/__snapshots__/Storyshots.test.js.snap

```diff
@@ -0,0 +1,80 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Storyshots Components/Button Login 1`] = `
```

Comment:
Sorry, I'm a little unfamiliar with how this works. Will this yield an actual image file?

Reply:
@junrezapico , snapshot is different on the visual regression testing, snapshot doesn't get an actual image instead it will gets the the codes initially and actually compares it with the existing codes in the code base when its running. This will not only compare the UI surface but also the behavior of the components as well.

so far this is what i have understand on the snapshot testing, its designed to lessen the pain on manual testing :)